core/api/src/main/java/com/blazebit/persistence/spi/JpaProvider.java

```diff
@@ -523,6 +523,14 @@
      */
     public void setCacheable(Query query);
 
+    /**
+     * Returns whether the JPA provider supports table group joins.
+     *
+     * @return true if supported, otherwise false
+     * @since 1.4.0
+     */
+    boolean supportsTableGroupJoins();
```

Comment:
I don't see a a reason for exposing this. Please remove it and place the relevant logic for hibernate in the `AbstractPersistenceTest` class of the hibernate testsuite base project.

Reply:
Im not really using it for anything but some test assumptions so I dont have a bunch of failing tests that shouldnt have been run for this category in the first place. I can remove the assumptions altogether, it just makes it a little bit harder to execute the tests outside of the Maven run profiles.

Reply:
You can still use the assumptions, just move the logic for `supportsTableGroupJoins` to `AbstractPersistenceTest`